### PR TITLE
CODE: Negative value

### PR DESCRIPTION
**Pull request by daikitag**
Extend the fixed value trait model to simulate effect sizes from opposing signs.

### CODE REVIEW

**Comment by daikitag** [comment]
I just want to quickly add this feature to enable simulation algorithm described by Barton et al. (2023).

File: tstrait/trait_model.py [hunk 1]
```diff
@@ -186,6 +186,10 @@ class TraitModelFixed(TraitModel):
     ----------
     value : float
         Value of the simulated effect size.
+    negative : bool, default False
```
Comment (jeromekelleher) [inline tstrait/trait_model.py:189]: I find ``negative`` a bit of a weird name. How about ``random_sign=False``?

**Review by jeromekelleher — COMMENTED** [review]
LGTM, but the parameter name is a bit weird.

File: tstrait/trait_model.py (hunk 1 above)
Comment (daikitag) [reply tstrait/trait_model.py]: I agree with that!

**Review by jeromekelleher — APPROVED** [review]

**Pull request merged**